src/runtime/vendor/github.com/kata-containers/govmm/qemu/qemu.go

```diff
@@ -2524,6 +2527,13 @@ func (config *Config) appendFDs(fds []*os.File) []int {
 	return fdInts
 }
 
+func (config *Config) appendSeccompSandbox() {
+	if config.SeccompSandbox != "" {
+		config.qemuParams = append(config.qemuParams, "-sandbox")
```

Comment:
The parameter name `-sandbox` seems a bit too generic. Why not just name it `-seccomp`?

Reply:
> The parameter name `-sandbox` seems a bit too generic. Why not just name it `-seccomp`?

This parameter is named by Qemu, not Kata. Kata just enable this feature by using Qemu's parameter. 
Please refer to this link for the qemu side: https://github.com/qemu/qemu/blob/7a9180b77eca258ab418ec00ab397392e70e55d9/qemu-options.hx#L3820